channeldb/payments.go

```diff
@@ -317,6 +344,124 @@ func deserializeOutgoingPayment(r io.Reader) (*OutgoingPayment, error) {
 	return p, nil
 }
 
+// CreationInfo is the information necessary to have ready when initiating a
+// payment, moving it into state InFlight.
+type CreationInfo struct {
```

Comment:
Not clear what the struct is from the name alone. It could use a `Payment` prefix, same goes for `AttemptInfo`.

Reply:
Fixed: https://github.com/lightningnetwork/lnd/pull/2761/commits/4b315c851b94c2d609e6bed1198632c849860a6a